skopt/optimizer/optimizer.py

```diff
@@ -152,6 +155,13 @@ def __init__(self, dimensions, base_estimator,
 
         self.n_jobs = n_jobs
 
+        # list of all strategies for parallelization. Used internally for testing.
+        # See description of `ask` method to see what parallel strategy means.
+        self.par_strats = ["cl_min", "cl_mean", "cl_max"]
```

Comment:
not a big fan of the name and we only use it once in `Optimizer` so can we define this as a tuple in the check that is done instead?